src/ontology/edits/oeo-physical.omn

```diff
@@ -3146,6 +3146,82 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/453",
         OEO_00000011
     
     
+Class: OEO_00010072
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An areal power density unit is a is unit that measures the power per surface area.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/597
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/615",
+        rdfs:label "areal power density unit"
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/UO_0000000>
+    
+    
+Class: OEO_00010073
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An areal energy density unit is a unit that measures the energy per surface area.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/597
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/615",
+        rdfs:label "areal energy density unit"
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/UO_0000000>
+    
+    
+Class: OEO_00010074
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An areal power density is a quantity value that measures the power per area.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "An areal power density is a quantity value that indicates the power on a certain area.",
```

Reply:
Quantity values and the corresponding units are not the same, there they need distinct definitions...

Reply:
Okay